### PR TITLE
fix: use matchBase option when match ignoreList

### PR DESCRIPTION
**Pull request by basecell**
I think use `matchBase` option is better.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1291,7 +1291,7 @@
                     "default": [],
                     "examples": [
                         [
-                            "**/*.js",
+                            "*.js",
```
Comment (JohnstonCode) [inline package.json:1294]: I would keep the `**/*.js` and just add another like `*.cache` or something.

Comment (basecell) [reply package.json:1294]: That's good, too. Shall I submit it?

Comment (JohnstonCode) [reply package.json:1294]: Yes Please

**Review by JohnstonCode — APPROVED** [review]

**Pull request merged**